projects/uniswap/index.js

```diff
@@ -1,38 +1,38 @@
-const { getChainTvl } = require('../helper/getUniSubgraphTvl');
 const sdk = require('@defillama/sdk')
-const {optimism} = require('../uniswapv3/index')
-
-const v1graph = getChainTvl({
-  ethereum: 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap'
-}, "uniswaps", "totalLiquidityUSD")
+const { getChainTvl } = require('../helper/getUniSubgraphTvl')
+const { optimism } = require('../uniswapv3/index')
+const { endpoints } = require('./api')
 
+const v1graph = getChainTvl(
+  {
+    ethereum: endpoints.v1,
+  },
+  'uniswaps',
+  'totalLiquidityUSD'
+)
 
 const v2graph = getChainTvl({
-  ethereum: 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2'
+  ethereum: endpoints.v2,
 })
 
-const v3Graphs = getChainTvl({
-  ethereum: "https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v3-subgraph",
-  optimism: "https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-optimism-dev",
-  arbitrum: 'https://api.thegraph.com/subgraphs/name/ianlapham/arbitrum-minimal',
-}, "factories", "totalValueLockedUSD")
+const v3Graphs = getChainTvl(endpoints.v3, 'factories', 'totalValueLockedUSD')
 
 module.exports = {
   timetravel: true,
   misrepresentedTokens: true,
   methodology: `Counts the tokens locked on AMM pools, pulling the data from the 'ianlapham/uniswapv2' subgraph`,
-  ethereum:{
-    tvl: sdk.util.sumChainTvls([v1graph("ethereum"), v2graph('ethereum'), v3Graphs('ethereum')]),
+  ethereum: {
+    tvl: sdk.util.sumChainTvls([v1graph('ethereum'), v2graph('ethereum'), v3Graphs('ethereum')]),
```

Comment:
in the current live version `v3Graphs('ethereum')` has been replaced with `v3Ethereum.tvl`, issue was that uniswap's v3 subgraph had problems syncing